frontend/src/types.ts

```diff
@@ -1946,6 +1946,8 @@ export interface FilterType {
     breakdowns?: Breakdown[]
     breakdown_group_type_index?: number | null
     breakdown_hide_other_aggregation?: boolean | null
+    /** @asType integer */
+    breakdown_limit?: number | null
```

Comment:
```suggestion
    breakdown_limit?: integer | null
```
We have a type alias for integers in the schema.

Reply:
Yeah, but not in the `types.ts` and if I export it, it totally changes all references in the schema.json 🙈  which is not ideal. Hence I just did it like this.

Reply:
Ah sorry, didn't see that this is `types.ts`. All good then!